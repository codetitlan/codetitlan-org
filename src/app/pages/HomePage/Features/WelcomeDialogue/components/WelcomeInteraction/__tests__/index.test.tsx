import * as React from 'react';
import { render } from '@testing-library/react';

import { WelcomeInteraction } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => ({})),
      },
    };
  },
}));

describe('<WelcomeInteraction  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<WelcomeInteraction />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
