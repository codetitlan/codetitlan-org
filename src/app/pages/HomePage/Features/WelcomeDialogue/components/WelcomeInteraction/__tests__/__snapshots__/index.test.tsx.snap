// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeInteraction  /> should match snapshot 1`] = `
.c0 {
  border: 1px solid #000000;
  margin: 3rem;
  padding: 1rem;
}

<div
  class="c0"
>
  <div
    class="Typewriter"
    data-testid="typewriter-wrapper"
  >
    <span
      class="Typewriter__wrapper"
    />
    <span
      class="Typewriter__cursor"
    >
      |
    </span>
  </div>
</div>
`;
